### ISSUE TITLE
Output size same as input size only if strides = 1

### ISSUE THREAD

**Issue by hannnwang** [tensorflow/tensorflow#48137]
## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D

## Description of issue (what needs changing):
In the description of the input argument "padding", it says:

_one of "valid" or "same" (case-insensitive). "valid" means no padding. "same" results in padding evenly to the left/right or up/down of the input such that output has the same height/width dimension as the input._

However, "same" would make the output have the same dimensions as the input only if strides is set to be 1. I was initially very confused by this typo.

**Comment by rmothukuru** [comment]
Created [a CL](https://critique-ng.corp.google.com/cl/365992643) which specifies the point about `strides=1`.
